pkg/vault/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Unreleased
+
+### New Features
+
+- `TimelockAuthorizer` contract.
+
+### Breaking Changes
+
+
```

Comment:
```suggestion
```

Don't worry about this for now, we don't yet have a plan for how to deal with these packages.